classes/shaman.lua

```diff
@@ -78,6 +98,7 @@ local RollingThunderHandler = {
         local hasSAO = not saoOption or type(saoOption[lightningShieldStacks]) == "nil" or saoOption[lightningShieldStacks];
         if (hasSAO) then
             local scale = 0.5 + 0.1 * (lightningShieldStacks - 6);
+            if SAO:IsCata() then local scale = 0.5 + 0.1 * (lightningShieldStacks - 5); end

```

Comment:
From 6 to 9 stacks, this will calculate the following scale factors:
- 6 stacks = 50%+10%\*(_nb_stacks_-5) = 50%+10%\*(6-5) = 50%+10% = 60%
- 7 stacks = 70%
- 8 stacks = 80%
- 9 stacks = 90%

Is that correct?

Reply:
changed to 0.4+0.1*(stacks-5) for Cata, so it'll be at 50% - 60% - 70% - 80%